graphene/src/matrix.rs

```diff
@@ -236,3 +236,45 @@ mod tests {
         );
     }
 }
+
+impl std::ops::Mul<Matrix> for Matrix {
+    type Output = Matrix;
+
+    fn mul(self, rhs: Matrix) -> Self::Output {
+        (&self).multiply(&rhs)
+    }
+}
+
+impl std::ops::Mul<Vec4> for Matrix {
```

Comment:
Could also add scalar multiplication here

Reply:
Did that in the last commit.